server/chat-plugins/wifi.tsx

```diff
@@ -1374,317 +1439,457 @@ function makePageHeader(user: User, pageid?: string) {
-						buf += `<br /><label for="winners">Number of winners: </label><input name="winners" /><br /><br />`;
-						buf += generatePokeballDropdown();
-						buf += `<br /><br /><label for="ivs">IVs (Formatted like "1/30/31/X/HT/30"): </label><input name="ivs" />`;
-						buf += `<br /><br /><label for="set">Prize:</label><br /><textarea style="width:70%;height:300px" placeholder="Paste set importable" name="set"></textarea>`;
-						buf += `<br /><br /><label for="info">Additional information (provide a link of proof here):</label><br /><textarea style="width:50%;height:100px" placeholder="Add any additional info" name="info"></textarea>`;
-						buf += `<br /><br /><button class="button" type="submit">Submit Lottery Giveaway</button>`;
-						buf += `</form>`;
-						break;
-					case 'question':
-						buf += `<form data-submitsend="/giveaway submit question {giver}|{ot}|{tid}|{game}|{question}|{answers}|{ivs}|{ball}|{info}|{set}">`;
-						buf += `<label for="giver">Giver:</label><input name="giver" /><br /><br />`;
-						buf += `<label for="ot">OT:</label><input name="ot" /><br /><br />`;
-						buf += `<label for="tid">TID:</label><input name="tid" /><br /><br />`;
-						buf += `Game:<div><input type="radio" id="bdsp" name="game" value="bdsp" checked /><label for="bdsp">BDSP</label><input type="radio" id="swsh" name="game" value="swsh" /><label for="swsh">SwSh</label></div>`;
-						buf += `<br /><label for="question">Question:</label><input name="question" /><br /><br />`;
-						buf += `<label for="answers">Answers (separated by comma):</label><input name="answers" /><br /><br />`;
-						buf += generatePokeballDropdown();
-						buf += `<br /><br /><label for="ivs">IVs (Formatted like "1/30/31/X/HT/30"): </label><input name="ivs" />`;
-						buf += `<br /><br /><label for="set"></label><textarea style="width:70%;height:300px" placeholder="Paste set importable here" name="set"></textarea>`;
-						buf += `<br /><br /><label for="info">Additional information (provide a link of proof here):</label><br /><textarea style="width:50%;height:100px" placeholder="Add any additional info" name="info"></textarea>`;
-						buf += `<br /><br /><button class="button" type="submit">Submit Question Giveaway</button>`;
-						buf += `</form>`;
-						break;
+						return <>
+							<h2>Submit a Giveaway</h2>
+							{(() => {
+								if (!type || !['question', 'lottery'].includes(type)) {
+									return <center>
+										<h3>Pick a giveaway type</h3>
+										{
+											formatFakeButton(`/view-giveaways-submitted-add-lottery`, <><i class="fa fa-random"></i> Lottery</>)
+										} | {
+											formatFakeButton(`/view-giveaways-submitted-add-question`, <><i class="fa fa-question"></i> Question</>)
+										}
+									</center>;
+								} else {
```

Comment:
Weird that eslint didn't catch this, but the `else` is unnecessary.

Reply:
did